<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSleepSessionDialog</class>
 <widget class="QDialog" name="AddSleepSessionDialog">
  <property name="windowTitle">
   <string>Add Sleep Session</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblStart">
       <property name="text">
        <string>Start Time:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateTimeEdit" name="dateTimeStart">
      </widget>
     </item>

     <item row="1" column="0">
      <widget class="QLabel" name="lblEnd">
       <property name="text">
        <string>End Time:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEnd"/>
     </item>

     <item row="2" column="0">
      <widget class="QLabel" name="lblQuality">
       <property name="text">
        <string>Quality (1-5):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinQuality">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>

     <item row="3" column="0">
      <widget class="QLabel" name="lblNotes">
       <property name="text">
        <string>Notes:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="textNotes"/>
     </item>
    </layout>
   </item>

   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
